e2e/process-services-cloud/process-task-attach-content-file-cloud.e2e.ts

```diff
@@ -0,0 +1,138 @@
+    const processDefinitionName = browser.params.resources.ACTIVITI_CLOUD_APPS.SIMPLE_APP.processes.uploadSingleMultipleFiles;
+    const uploadWidgetId = browser.params.resources.ACTIVITI_CLOUD_APPS.SIMPLE_APP.forms.uploadSingleMultiple.widgets.contentMultipleAttachFileId;
+    const taskName = browser.params.resources.ACTIVITI_CLOUD_APPS.SIMPLE_APP.tasks.uploadSingleMultipleFiles;
+    const processName = StringUtil.generateRandomString(10);
+    const folderName = StringUtil.generateRandomString(5);
+    let uploadedFolder: any;
+
+    const pdfFileOne = {
+        'name': browser.params.resources.Files.ADF_DOCUMENTS.PDF.file_name,
+        'location': browser.params.resources.Files.ADF_DOCUMENTS.PDF.file_location
+    };
+
+    const pdfFileTwo = {
+        'name': browser.params.resources.Files.ADF_DOCUMENTS.PDF_B.file_name,
+        'location': browser.params.resources.Files.ADF_DOCUMENTS.PDF_B.file_location
+    };
+
+    this.alfrescoJsApi = new AlfrescoApi({
+        provider: 'ECM',
+        hostEcm: browser.params.config.bpmHost
+    });
+    const uploadActions = new UploadActions(this.alfrescoJsApi);
+
+    beforeAll(async () => {
+        await loginSSOPage.loginSSOIdentityService(browser.params.testConfig.hrUser.email, browser.params.testConfig.hrUser.password);
+
+        await this.alfrescoJsApi.login(browser.params.testConfig.hrUser.email, browser.params.testConfig.hrUser.password);
+        uploadedFolder = await uploadActions.createFolder(folderName, '-my-');
+        await uploadActions.uploadFile(pdfFileOne.location, pdfFileOne.name, uploadedFolder.entry.id);
+        await uploadActions.uploadFile(pdfFileTwo.location, pdfFileTwo.name, uploadedFolder.entry.id);
+        await navigationBarPage.navigateToProcessServicesCloudPage();
+        await appListCloudComponent.checkApsContainer();
+    });
+
+    it('[C311290] Should be able to attach multiple files when widget allows multiple files to be attached from content', async () => {
+        await appListCloudComponent.checkAppIsDisplayed(simpleApp);
+        await appListCloudComponent.goToApp(simpleApp);
+        await processCloudDemoPage.openNewProcessForm();
+        await startProcessPage.startProcessWithProcessDefinition(processName, processDefinitionName);
+
+        await processCloudDemoPage.processFilterCloudComponent.clickOnProcessFilters();
+        await processCloudDemoPage.processFilterCloudComponent.clickRunningProcessesFilter();
+        await expect(await processCloudDemoPage.processFilterCloudComponent.getActiveFilterName()).toBe('Running Processes');
+        await processCloudDemoPage.processListCloudComponent().checkContentIsDisplayedByName(processName);
+        await processCloudDemoPage.processListCloudComponent().selectRow(processName);
+
+        await tasksCloudDemoPage.taskListCloudComponent().checkTaskListIsLoaded();
+        await tasksCloudDemoPage.taskListCloudComponent().selectRow(taskName);
```

Comment:
I would suggest to create the process instance using API call and navigate directly to the task created.

Reply:
it makes sense. @sivakumar414ram

Reply:
Thanks.  i will update it

Reply:
Updated e2e (now process instance is getting created using API).